service/src/main/java/bio/terra/pipelines/app/configuration/internal/StairwayDatabaseConfiguration.java

```diff
@@ -7,7 +7,7 @@
 
 @Configuration
 @EnableConfigurationProperties
-@ConfigurationProperties(prefix = "pipelines.stairway-database")
+@ConfigurationProperties(prefix = "spring.stairway-database")
```

Comment:
~would it make sense to put this under `"stairway.stairway-database"` instead?~ nvm i think this makes sense with the other things we have under "spring"